contracts/metatx/MinimalForwarder.sol

```diff
@@ -69,4 +76,11 @@ contract MinimalForwarder is EIP712 {
 
         return (success, returndata);
     }
+
+    function _verify(ForwardRequest calldata req, bytes calldata signature) internal view returns (bool, bool) {
+        address signer = _hashTypedDataV4(
+            keccak256(abi.encode(_TYPEHASH, req.from, req.to, req.value, req.gas, req.nonce, keccak256(req.data)))
+        ).recover(signature);
+        return (_nonces[req.from] == req.nonce, signer == req.from);
```

Comment:
The two bool that are returned are never used independently. What was the rational for doing that instead of returning a "valid" boolean?
```suggestion
        return _nonces[req.from] == req.nonce && signer == req.from;
```

Reply:
I was playing with the idea of having different errors depending on whether the derived signer wasn't the expected one or the nonce wasn't the expected, but didn't go well. Thanks for noticing! I'm reverting back to the original behavior of a single boolean